geopandas/_vectorized.py

```diff
@@ -885,6 +886,19 @@ def normalize(data):
         return out
 
 
+def make_valid(data):
+    if compat.USE_PYGEOS:
+        return pygeos.make_valid(data)
+    else:
```

Comment:
```suggestion
    else:
        if not SHAPELY_GE_18:
            raise ImportError(
                f"shapely >= 1.8 or PyGEOS is required, version {shapely.__version__} is installed"
            )
```

Since `make_valid` has been introduced in shapely 1.8, we need to raise an informative warning here if a user tries to use it with shapely 1.7. Something like this, though the line will likely be too long for black.

Reply:
committed